Svg import broken

_I am..._

| -------------------------------------------------------------------------------------------------
| Reporting a bug or issue
| -------------------------------------------------------------------------------------------------

**Expected behavior:**

![image](https://user-images.githubusercontent.com/6757532/81488125-afa9aa80-925c-11ea-9eb8-8b94061e9d6c.png)


**Observed behavior:**

```sh
> basic-svg@1.0.0 postinstall /Users/<redacted>/basic-svg
> npm run build && skpm-link


> basic-svg@1.0.0 build /Users/<redacted>/basic-svg
> skpm-build

[1/2] 🖨  Copied src/manifest.json in 3ms
error Error while building ./my-command.js
./node_modules/node-gyp-build/index.js
Module not found: Error: Can't resolve 'fs' in '/Users/<redacted>/basic-svg/node_modules/node-gyp-build'
resolve 'fs' in '/Users/<redacted>/basic-svg/node_modules/node-gyp-build'
  Parsed request is a module
  using description file: /Users/<redacted>/basic-svg/node_modules/node-gyp-build/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/<redacted>/basic-svg/node_modules/node-gyp-build/package.json (relative path: .)
    resolve as module

```

**How to reproduce:**

```bash
curl https://codeload.github.com/airbnb/react-sketchapp/tar.gz/master | tar -xz --strip=2 react-sketchapp-master/examples/basic-svg
cd basic-svg
```

**Sketch version:**

`63.1`

**Please attach ~screenshots, a zip file of your project, and~ or a link to your github project**

> https://github.com/airbnb/react-sketchapp/tree/master/examples/basic-svg

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/). Every release, along with the migration instructions, is documented on the Github [Releases](https://github.com/airbnb/react-sketchapp/releases) page.
 
+## Version 3.2.5
+
+- Fix Skpm taking the wrong entry point when requiring react-sketchapp
+
 ## Version 3.2.4
 
 - Fix the generated ES package

--- a/examples/basic-svg/package.json
+++ b/examples/basic-svg/package.json
@@ -16,7 +16,7 @@
   "author": "Jon Gold <jon.gold@airbnb.com>",
   "license": "MIT",
   "devDependencies": {
-    "@skpm/builder": "^0.4.0"
+    "@skpm/builder": "^0.7.5"
   },
   "dependencies": {
     "prop-types": "^15.5.8",

--- a/package.json
+++ b/package.json
@@ -5,7 +5,6 @@
   "sideEffects": false,
   "main": "lib/index.js",
   "module": "lib/module/index.js",
-  "sketch": "lib/module/index.sketch.js",
   "types": "lib/index.d.ts",
   "license": "MIT",
   "author": "Jon Gold <jon.gold@airbnb.com> (http://jon.gold)",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-sketchapp",
-  "version": "3.2.4",
+  "version": "3.2.5",
   "description": "A React renderer for Sketch.app",
   "sideEffects": false,
   "main": "lib/index.js",

--- a/src/entrypoint.sketch.ts
+++ b/src/entrypoint.sketch.ts
@@ -37,5 +37,3 @@ export function makeSymbol(
 }
 
 export const TextStyles = _TextStyles(() => SketchBridge);
-
-export * from './index.common';

--- a/src/entrypoint.ts
+++ b/src/entrypoint.ts
@@ -1,0 +1,44 @@
+import { render as _render } from './render';
+import { renderToJSON as _renderToJSON } from './renderToJSON';
+import { makeSymbol as _makeSymbol, SymbolMasterProps } from './symbol';
+
+import {
+  SketchLayer,
+  WrappedSketchLayer,
+  SketchDocumentData,
+  WrappedSketchDocument,
+  SketchDocument,
+  PlatformBridge,
+} from './types';
+
+import { TextStyles as _TextStyles } from './sharedStyles/TextStyles';
+
+function getDefaultPlatformBridge() {
+  return require('./platformBridges/macos').default;
+}
+
+export function render(
+  element: React.ReactElement,
+  container?: SketchLayer | WrappedSketchLayer,
+  platformBridge: PlatformBridge = getDefaultPlatformBridge(),
+) {
+  return _render(platformBridge)(element, container);
+}
+
+export function renderToJSON(
+  element: React.ReactElement,
+  platformBridge: PlatformBridge = getDefaultPlatformBridge(),
+) {
+  return _renderToJSON(platformBridge)(element);
+}
+
+export function makeSymbol(
+  Component: React.ComponentType<any>,
+  symbolProps: string | SymbolMasterProps,
+  document: SketchDocumentData | SketchDocument | WrappedSketchDocument | undefined,
+  bridge: PlatformBridge = getDefaultPlatformBridge(),
+) {
+  return _makeSymbol(bridge)(Component, symbolProps, document);
+}
+
+export const TextStyles = _TextStyles(() => getDefaultPlatformBridge());

--- a/src/index.common.ts
+++ b/src/index.common.ts
@@ -1,6 +1,0 @@
-export { Platform } from './Platform';
-export { StyleSheet } from './stylesheet';
-export { getSymbolComponentByName, getSymbolMasterByName, injectSymbols } from './symbol';
-export { useWindowDimensions } from './context';
-
-export * from './components';

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,46 +1,7 @@
-import { render as _render } from './render';
-import { renderToJSON as _renderToJSON } from './renderToJSON';
-import { makeSymbol as _makeSymbol, SymbolMasterProps } from './symbol';
+export { Platform } from './Platform';
+export { StyleSheet } from './stylesheet';
+export { getSymbolComponentByName, getSymbolMasterByName, injectSymbols } from './symbol';
+export { useWindowDimensions } from './context';
 
-import {
-  SketchLayer,
-  WrappedSketchLayer,
-  SketchDocumentData,
-  WrappedSketchDocument,
-  SketchDocument,
-  PlatformBridge,
-} from './types';
-
-import { TextStyles as _TextStyles } from './sharedStyles/TextStyles';
-
-function getDefaultPlatformBridge() {
-  return require('./platformBridges/macos').default;
-}
-
-export function render(
-  element: React.ReactElement,
-  container?: SketchLayer | WrappedSketchLayer,
-  platformBridge: PlatformBridge = getDefaultPlatformBridge(),
-) {
-  return _render(platformBridge)(element, container);
-}
-
-export function renderToJSON(
-  element: React.ReactElement,
-  platformBridge: PlatformBridge = getDefaultPlatformBridge(),
-) {
-  return _renderToJSON(platformBridge)(element);
-}
-
-export function makeSymbol(
-  Component: React.ComponentType<any>,
-  symbolProps: string | SymbolMasterProps,
-  document: SketchDocumentData | SketchDocument | WrappedSketchDocument | undefined,
-  bridge: PlatformBridge = getDefaultPlatformBridge(),
-) {
-  return _makeSymbol(bridge)(Component, symbolProps, document);
-}
-
-export const TextStyles = _TextStyles(() => getDefaultPlatformBridge());
-
-export * from './index.common';
+export * from './components';
+export * from './entrypoint';

